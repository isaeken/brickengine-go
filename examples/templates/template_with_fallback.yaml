disk_size: {{ vars.disk_size | 1024 }}
hostname: {{ slug(vars.hostname) | "fallback-host" }}
