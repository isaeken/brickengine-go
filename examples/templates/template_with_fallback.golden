disk_size: 1024
hostname: fallback-host
